pkg/odo/cli/debug/portforward.go

```diff
@@ -166,7 +165,7 @@ func (o PortForwardOptions) Run() error {
 		return err
 	}
 
-	return o.PortForwarder.ForwardPorts(o.PortPair, o.StopChannel, o.ReadyChannel, o.isExperimental)
+	return o.PortForwarder.ForwardPorts(o.PortPair, o.StopChannel, o.ReadyChannel, util.CheckPathExists(o.devfilePath))
```

Comment:
maybe store `util.CheckPathExists(o.devfilePath)` in a variable and pass here

Reply:
Since we're only retrieving this variable once, I think it'd be okay to pass in util.CheckPathExists to ForwardPorts since ForwardPorts is using the parameter `isS2i` which requires true / false.

Reply:
Actually we are retrieving this twice. First at complete() and then here. Maybe store in a variable like isDevfile once like in the other files and pass it to the function.